src/assets/main.js

```diff
@@ -1,10 +1,6 @@
-let baseUrl = window.location.pathname;
-let fileIndex = baseUrl.indexOf("index.html");
-if (fileIndex > 0) {
-    baseUrl = baseUrl.slice(0, fileIndex);
-}
+let baseUrl = window.location.origin;
```

Comment:
I think it's okay for now. On the other hand if we have a page such as https://origin/some/other/nested/page.html the assets should still be at https://origin/assets so then `window.location.origin` is the right path